external/fv3fit/fv3fit/_shared/config.py

```diff
@@ -282,9 +282,13 @@ def slice(self):
 ClipConfig = Mapping[Hashable, Mapping[str, SliceConfig]]
 
 
+def _clip_config_factory() -> ClipConfig:
+    return {}
+
+
 @dataclasses.dataclass(frozen=True)
 class PackerConfig:
-    clip: ClipConfig
+    clip: ClipConfig = dataclasses.field(default_factory=_clip_config_factory)
```

Comment:
What you had before was correct, I was just thrown off by ClipConfig not being a dataclass. As we discussed in slack, please rename ClipConfig to anything without Config in the name.